addon/ng2/blueprints/enum/files/__path__/__name__.ts

```diff
@@ -0,0 +1,3 @@
+export enum <%= classifiedModuleName %> {
+
+}
```

Comment:
Please change this filename to `__name__.enum.ts` this will make it consistent with other file names.